src/frontend/screens/Game/GamePage/components/DotsMenu.tsx

```diff
@@ -68,24 +72,22 @@ const DotsMenu = ({ gameInfo, handleUpdate }: Props) => {
         </Dialog>
       )}
 
-      {showDlcs && (
-        <Dialog showCloseButton onClose={() => setShowDlcs(false)}>
-          <DialogHeader onClose={() => setShowDlcs(false)}>
-            <div>{t('game.dlcs', 'DLCs')}</div>
-          </DialogHeader>
-          <DialogContent>
-            {gameInstallInfo ? (
-              <DLCList
-                dlcs={gameInstallInfo?.game.owned_dlc}
-                runner={runner}
-                mainAppInfo={gameInfo}
-                onClose={() => setShowDlcs(false)}
-              />
-            ) : (
-              <UpdateComponent inline />
-            )}
-          </DialogContent>
-        </Dialog>
+      {showModifyInstallModal && (
+        <ModifyInstallModal
+          gameInfo={gameInfo}
+          gameInstallInfo={gameInstallInfo}
+          onClose={() => setShowModifyInstallModal(false)}
+        />
+      )}
```

Comment:
Modify Install Modal completely replaces DLCs modal, although for Epic games it's still the same component under the hood